dictionary.json

```diff
@@ -48,6 +48,35 @@
       "description": "The logging provider or logging service that logged the event. For example, Microsoft-Windows-Security-Auditing.",
```

Comment:
A comment for my own curiosity/understanding. What function does the dictionary.json file serve?

Reply:
Sure, that is a good question. The attribute dictionary serves as the repository for all available attributes within the OCSF. The attributes defined by the dictionary constitute the fundamental components of OCSF objects and classes. For example, within this framework, an object represents a specific collection of attributes drawn from the dictionary. Similarly, an event class represents a collection of attributes and objects.